packages/components/src/components/modal/modal.jsx

```diff
@@ -97,9 +101,14 @@ const ModalElement = ({
             {(header || title || rendered_title) && (
                 <div
                     className={classNames('dc-modal-header', {
+                        'dc-modal-header__border-bottom': !should_header_stick_body,
                         [`dc-modal-header--${className}`]: className,
                         [`dc-modal-header--is-title-centered`]: is_title_centered,
+                        [`dc-modal-header--is-item-align-centered`]: !is_align_header_content_left,
```

Comment:
```suggestion
                        [`dc-modal-header--is-item-align-left`]: is_align_header_content_left,
```

Reply:
@mehrzad-fs since it resulted in lots of confusion from the first of this PR, I removed this condition from `modal` and handled it in my `popup` component.